extensions/default/src/getSopClassHandlerModule.js

```diff
@@ -29,6 +29,7 @@ const makeDisplaySet = instances => {
     SeriesDescription: instance.SeriesDescription || '',
     Modality: instance.Modality,
     isMultiFrame: isMultiFrame(instance),
+    countIcon: displayReconstructableInfo.value ? 'icon-mpr' : undefined,
```

Comment:
Is this really a "count" icon? It seems to be more related to the display set/series itself, no? Maybe just `icon`?

Reply:
It isn't just icon, because that would be the overall icon - the default icon IS a count indicator, so I think countIcon is ok.